config/opal_check_pc.m4

```diff
@@ -0,0 +1,40 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2021 IBM Corporation.  All rights reserved.
+dnl
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+AC_DEFUN([OPAL_CHECK_LFLAGS], [
+         AC_MSG_CHECKING([for pc file for])
+         AC_MSG_RESULT([$1])
+
+	 num_paths=$#
+         save_name=$1
+         cmd="pkg-config"
+         $1_OMPI_PC_DIR=""
```

Comment:
You're assuming that $1 is a m4 literal.  That's fine (ie, you don't have to fix that), but assert that it is a literal earlier in the macro so we're not surprised.  Something like:

```
AS_LITERAL_WORD_IF([$1], [], m4_fatal([Non-literal argument $1])])
```

That's completely untested, of course, but I think it's right.

Reply:
Thanks, added.